src/fprime/fbuild/cmake.py

```diff
@@ -439,19 +452,30 @@ def cmake_refresh_cache(self, build_dir, full=False):
                 run_args,
                 write_override=True,
                 environment=environment,
-                print_output=self.verbose,
+                print_output=True,
             )
         else:
             if self.verbose:
                 print("[CMAKE] Checking CMake cache for rebuild")
-            self.execute_known_target(
-                "noop",
-                build_dir,
-                None,
-                top_target=True,
-                full_cache_rebuild=True,
-                print_output=self.verbose,
-            )
+            # Backwards compatibility: refresh_cache was named noop until v3.3.x
+            if self.is_target_supported(build_dir, "noop"):
```

Comment:
Given the slowness of printing targets, perhaps it is faster to run "refresh_cache" and trap "No rule to make target `llkjldksjlksdj'.  Stop." and then run "noop"?

Reply:
Will revisit with the ninja support work.